### ISSUE TITLE
Hey!

### ISSUE THREAD

**Issue by trusktr** [ZoeDreams/confident-ai#1]
Just noticed you started this project. Using AI + Graphics sounds interesting! I wonder what we can do.

**Comment by p3nGu1nZz** [comment]
yes i need to circle back to this. I went pretty deep during my first pass at this. I successfully got local speech detection for specific wake words using pocketspinx that runs locally. However it works not as good as Alexa or other cloud services.. I was also using a very large dictionary for the possible wake words, which I think is incorrect. For initial set up I have it working with googles speech detection, and text to speech. 

The other aspect to this is to get a transformer running locally which can provide the intelligence behind what and how to respond to the speech to text that was input.

**Comment by p3nGu1nZz** [comment]
I might roll this project up with the API Iwas writing for openai, because I had some ideas on how to dynamically train the model based on long short term memory buffers using a recurrent temporal deep neural net. However I really am more interested in how to get stuff to work localized. check out this stuff. https://huggingface.co/models?pipeline_tag=text-to-speech&sort=downloads
